docs/dev/api_integration.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Create an API Integration
+type: documentation
+---
+
+## Overview
+
+You can use [Datadog API endpoints][1] to programmatically send data to Datadog, manage your account, and visualize data collected from an API integration. 
+
+API integrations send the following types of data to Datadog:
+
+- [Metrics][12]
+- [Logs & Log Pipelines][13]
+- [Events][14]
+- [Service Checks][15]
+- [Traces][16]
+- [Incidents][17]
+- [Security Events][18]
+
+This page provides instructions for creating an API integration in the `integrations-extras` repository. For more information about why you would want to create an API-based integration, see [Creating your own solution][2].
+
+## Setup
+
+### Prerequisites
+
+- You must have an [API key][3] and [application key][4].
+- Determine which [Datadog site][5] you want to use.
+
+An API key is required to submit data to a Datadog API endpoint. An application key is required to query data from Datadog or to create resources within the Datadog site. For more information, see [API and Application Key][6].
+
+Create a connection to Datadog in your company's platform using the API key, application key, and site URL. 
+
+### Create an OAuth client
+
+Datadog recommends setting up an [OAuth client][7] to securely authorize third-party application access in your integrations. For more information, see [OAuth for Integrations][8] and [Authorization Endpoints][9].
```

Comment:
```suggestion
### Prerequisites

- You must have an [API key][3] and [application key][4].
- Determine which [Datadog site][5] you want to use.

An API key is required to submit data to a Datadog API endpoint. An application key is required to query data from Datadog or to create resources within the Datadog site. For more information, see [API and Application Key][6].

Create a connection to Datadog in your company's platform using the API key, application key, and site URL. 

### Create an OAuth client
Instead of requesting these credentials directly from a user, Datadog recommends using an [OAuth client][7] to handle authorization and access for API-based integrations. For more information, see [OAuth for Integrations][8] and [Authorization Endpoints][9]

You can explore examples of existing API integrations in the `integrations-extras` repository such as [Vantage][23].
```